sync2jira/upstream_issue.py

```diff
@@ -358,6 +383,22 @@ def github_issues(upstream, config):
         if issue.get('milestone', None):
             issue['milestone'] = issue['milestone']['title']
 
+        if not issue.get('storypoints', None):
+            issue['storypoints'] = ''
+            orgname, reponame = upstream.rsplit('/', 1)
+            issuenumber = issue['number']
+            github_project_fields = config['sync2jira']['map']['github'][upstream]['github_project_fields']
+            variables = {"orgname": orgname, "reponame": reponame, "issuenumber": issuenumber}
+            for fieldname, values in github_project_fields.items():
+                variables['ghfieldname'] = values[0]
```

Comment:
I don't know how much value this adds, but writing it this way might help future readers understand what the structure of that tuple is supposed to be more easily.

```suggestion
                ghfieldname, jirafieldname = values
                variables['ghfieldname'] = ghfieldname
```

Reply:
I have adapted the suggestion from @ralphbean with a small change. b921b55434d2599f3c7462d25f84c52fde66f305 Ignored the second variable as it is not used anywhere.